### PR TITLE
feat: add Full Stack Europe 2023

### PR DESCRIPTION
**Pull request by fmerian**
this commit also updates several closed CFP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,6 +231,7 @@ All the events (past and coming) are available publicly in JSON: https://develop
 * 10-12: [SREcon23 Europe/Middle East/Africa](https://www.usenix.org/srecon) - Dublin (Ireland)
 * 12: [Cloud Nord](https://www.cloudnord.fr/) - Lille (France)
 * 12-13: [Volcamp 2023](https://www.volcamp.io/) - Clermont-Ferrand (France)
+* 18-20: [Full Stack Europe](https://fullstackeurope.com/2023) - Antwerp (Belgium) <a href="[https://sessionize.com/wearedevelopers-worldcongress23/](https://docs.google.com/forms/d/e/1FAIpQLSdgQjf39QepP07gLE88g7-Cn7Xdfast_mHVSUp25CfpjAM2uw/viewform)"><img alt="CFP Full Stack Europe 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2030-June-2023&color=green"></a>
```
Comment (scraly) [inline README.md:234]:
```suggestion
* 18-20: [Full Stack Europe](https://fullstackeurope.com/2023) - Antwerp (Belgium) <a href="https://docs.google.com/forms/d/e/1FAIpQLSdgQjf39QepP07gLE88g7-Cn7Xdfast_mHVSUp25CfpjAM2uw/viewform"><img alt="CFP Full Stack Europe 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2030-June-2023&color=green"></a>
```

**Review by scraly — CHANGES_REQUESTED** [review]
thanks,
just a fix please :)

**Comment by fmerian** [comment]
Oops, fixed. Thanks, @scraly!

**Comment by scraly** [comment]
thanks :)

**Pull request merged**